src/libsyntax/errors/emitter.rs

```diff
@@ -663,7 +663,7 @@ fn stderr_isatty() -> bool {
     type DWORD = u32;
     type BOOL = i32;
     type HANDLE = *mut u8;
-    const STD_ERROR_HANDLE: DWORD = -12i32 as DWORD;
+    const STD_ERROR_HANDLE: DWORD = -12 as DWORD;
```

Comment:
This suffix is necessary since `DWORD` is unsigned on windows. (or, I think that's the reason)

cc @retep998 

http://buildbot.rust-lang.org/builders/auto-win-msvc-64-opt/builds/3376/steps/compile/logs/stdio